state/services.go

```diff
@@ -9,7 +9,7 @@ import (
 // NOTE: Interfaces used by RPC must be thread safe!
 //------------------------------------------------------
 
-//go:generate mockery --case underscore --name BlockStore
+//go:generate go run github.com/vektra/mockery/v2 --disable-version-string --case underscore --name BlockStore
```

Comment:
This occurs in enough places that I would think about maybe checking in a script like

```sh
cat >scripts/generate_type.sh <<EOF
#!/bin/sh
go run github.com/vektra/mockery/v2 --disable-version-string --case underscore --name $*
EOF
```

or words to that effect. Then these could become 

```go
//go:generate ../scripts/generate_type.sh BlockStore
```

and so on. Beyond just being shorter, this also has the advantage that it's easier to be sure we didn't leave out a flag somewhere, and to update all the usage sites consistently when the options change. YMMV.